R/performance_roc.R

```diff
@@ -122,6 +123,21 @@ print.performance_roc <- function(x, ...) {
 }
 
 
+#' @export
+as.numeric.performance_roc <- function(x, ...) {
+  if (length(unique(x$Model)) == 1) {
+    auc <- bayestestR::area_under_curve(x$Specificity, x$Sensitivity)
+  } else {
+    dat <- split(x, f = x$Model)
+
+    auc <- c()
+    for (i in seq_along(dat)) {
+      auc <- c(auc, bayestestR::area_under_curve(dat[[i]]$Specificity, dat[[i]]$Sensitivity))
+    }
+  }
+  auc
+}
```

Comment:
I don't know if you expect many models to be passed in `x` but if so, it will be more performant to preallocate the vector instead of extending it in the loop:

```r
auc <- numeric(length(dat))
for (i ...) {
  auc[i] <- bayestestR::...
}
```